tests/testthat/test-install.R

```diff
@@ -327,7 +327,10 @@ test_that("install() prefers cellar when available", {
   expect_equal(record$Source, "Cellar")
 
   prefix <- if (renv_platform_windows()) "file:///" else "file://"
-  uri <- paste0(prefix, renv_tests_path("local/skeleton"))
+  uri <- normalizePath(
```

Comment:
What are we using `normalizePath()` here for? (Just feels awkward because this is a file URI not a path)

Reply:
Ooops, I should normalize the path of the url. It's my stab in the dark to make a test pass on windows. I think prior to #1272, the normalization was taken care of by someone else.